config/globalkeyshortcuts.conf

```diff
@@ -31,11 +31,11 @@ Enabled=true
 Exec=pcmanfm-qt
 
 [Control%2BAlt%2BI.7]
-Comment=Internet. Web browser.
+Comment=Web browser
 Enabled=true
-Exec=xdg-open, http://google.com
+Exec=x-www-browser
```

Comment:
x-www-browser is debian and derivatives only (inkluding Ubuntu)
so the xdg thing is better imho